### PR TITLE
Add a Docker image with pre-commit installed

### PR DESCRIPTION
**Pull request by vulkoingim**
## Description

This fixes the pre-commit not found

## How to test

Open a workspace from this commit and try to commit something or run `pre-commit run --all-files`

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,15 +1,11 @@
-image: gitpod/workspace-full:2022-08-04-13-40-17
+image: eu.gcr.io/gitpod-core-dev/dev-environment:observability-2022-08-15
```
Comment (ArthurSens) [inline .gitpod.yml]:
```suggestion
image: 
  file: gitpod.Dockerfile
```

**Review by ArthurSens — COMMENTED** [review]
Just one small suggestion so we don't need to update the image URL everytime we change our dockerfile 🙂 

Here is the reference: https://www.gitpod.io/docs/references/gitpod-yml#imagefile

File: .gitpod.yml (hunk 1 above)
Comment (vulkoingim) [reply .gitpod.yml]: Ooh nice :)

File: .gitpod.yml [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
+image: 
+  file: gitpod.Dockerfile
 workspaceLocation: observability/observability.code-workspace
 tasks:
-  - init: |
-      # TODO(arthursens): We should update the workspace image so we dont need the steps below
-      sudo curl -Lo /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 && sudo chmod +x /usr/bin/yq
-      sudo apt update ; sudo apt-get install shellcheck apt-transport-https ca-certificates -y ; sudo update-ca-certificates
+  - name: Install pre-commit hooks & generate manifests
+    command: |
```
Comment (ArthurSens) [inline .gitpod.yml:6]:
```suggestion
    init: |
```

Sorry, I've also missed this. I think prebuilds are still very important to have... although they are a bit unstable at the moment, it is one of the core functionalities of Gitpod. We should dogfood our own product and provide feedback when things don't work as needed 😅.

Now that I finally joined the Gitpod team I can see [why our prebuilds](https://gitpod.io/t/gitpod/observability/prebuilds) were taking so much time and looks like our own setup is broken 😬

Comment (vulkoingim) [reply .gitpod.yml:6]: Yeah, I forgot to comment on that.

I would suggest tackling this in another issue so at least the WS works until we fix the prebuilds

File: .gitpod.yml [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
       make --always-make
```
Comment (ArthurSens) [inline .gitpod.yml:7]:
```suggestion
```

Totally unrelated to your PR, but this is the reason our prebuilds are failing.

We use `go get` to install a bunch of tools and we broke that when we updated the image to go 1.19. We can use `go install` instead but those tools were very focused on jsonnet... we don't really need it.

I'll open another PR removing this line

**Review by ArthurSens — APPROVED** [review]

**Pull request merged**